config/initializers/haml.rb

```diff
@@ -1 +0,0 @@
-Haml::Template.options[:ugly] = true
```

Comment:
this was deprecated. keeping the init file in case -- not sure what the convention here is.